src/main.rs

```diff
@@ -29,6 +29,9 @@ async fn main() {
             });
         }
     }
+    if users.len() == 0 {
+        return;
```

Comment:
Could you change it to still post a message, such as "It's nobody's birthday today :sad-bidoof:" ? We want to ensure it still runs and doesn't just fail

Reply:
"Happy Birthday to no one. :sad-bidoof:"